sys/net/gnrc/routing/rpl/gnrc_rpl.c

```diff
@@ -45,6 +48,7 @@ kernel_pid_t gnrc_rpl_init(kernel_pid_t if_pid)
 {
     /* check if RPL was initialized before */
     if (gnrc_rpl_pid == KERNEL_PID_UNDEF) {
+        _instance_id = 0;
```

Comment:
how about introduce and use a define `RPL_DEFAULT_INSTANCE` with value of 0 [1]?
(just a question, I have no strong opinion on it)

[1] https://tools.ietf.org/html/rfc6550#section-17